node/core/approval-voting/src/tests.rs

```diff
@@ -332,6 +333,60 @@ impl Backend for TestStore {
 	}
 }
 
+#[derive(Clone)]
+pub(super) struct SharedTestStore<T: Backend> {
```

Comment:
I wonder if this needs to be generic. After all, in test code every type should be known. Or is this used with multiple different kinds of backends?

Reply:
If you feel this should still be a specific type, I can address that as well.